workspace_tools/targets.py

```diff
@@ -1089,6 +1089,16 @@ def program_cycle_s(self):
         return 2
 
 
+### Maxim Integrated ###
+
+class MAXWSNENV(Target):
+    def __init__(self):
+        Target.__init__(self)
+        self.core = "Cortex-M3"
+        self.extra_labels = ['Maxim', 'MAX32610']
+        self.macros = ['__SYSTEM_HFX=24000000']
+        self.supported_toolchains = ["GCC_ARM", "IAR", "ARM"]
+        self.default_toolchain = "GCC_ARM"
```

Comment:
Here's the link for explanation : http://www.keil.com/support/man/docs/armlib/armlib_chr1358938938431.htm